src/benchmarks/micro/CommandLineOptions.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+
+namespace MicroBenchmarks
+{
+    class CommandLineOptions
```

Comment:
could you please move all the parsing logic to `BenchmarkDotNet.Extensions` project? the goal is to keep all the utilities in this helper library while having only benchmarks in `benchmarks/*`

Also if possible it would be great to add a simple test to `src/tests/harness/BenchmarkDotNet.Extensions.Tests`

Reply:
Done